en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -0,0 +1,52 @@
+# Multi-Vehicle Simulation with JMAVSim
+
+This topic explains how to simulate multiple UAV (multicopter) vehicles using JMAVSim and SITL.
+
+> **Tip** This is the easiest way to simulate multiple vehicles running PX4. 
+  It works on PX4 v1.8.0 and later.
+
+## Instances on Separate Ports
+
+To start multiple instances on separate ports:
+
+1. Build PX4
+   ```
+   make posix_sitl_default
+   ```
+1. Run **sitl_multiple_run.sh**, specifying the number of instances to start (e.g. 2):
+   ```
+   ./Tools/sitl_multiple_run.sh 2
+   ```
+1. Start the first instance:
+   ```
+   ./Tools/jmavsim_run.sh
+  ```
+1. Start subsequent instances, specifying the UDP port for the instance:
+   ```
+   ./Tools/jmavsim_run.sh -p 14561
+   ```
+   The port should be set to `14560+i` for `i` in `[0, N-1]`.
+
+## Instances on Same Port
```

Comment:
I'm not sure if we should add this. It's generally not something that you would want, and should there be a need for it, we can add a simpler way to achieve this.

Reply:
We should find a method for this ideally. The use case is multiple devices attached to a real Wifi network. The test case is Dronecode SDK, which in theory supports this.